### DISCUSSION TITLE
useExternal

### DISCUSSION THREAD

**Discussion by wangcongyi** [alibaba/hooks#1275] [Q&A]
`useExternal`  的  101~ 106 行   
action 方法使用了 useMemo ，
依赖项为什么是 `setActive`

**Comment by brickspert** [comment]
确实应该删掉，笔误。

**Reply by wangcongyi** [reply]
[history](https://github.com/alibaba/hooks/commit/5013bb3302c04fbdb0234fc912eede0ea83b45f3#diff-17c3d635dccbe99859963520f80bf8a501872e4fd7e8f3864d5a53d10b3d99d2)
